Content.Client/Chat/UI/EmotesMenu.xaml.cs

```diff
@@ -36,7 +38,7 @@ public EmotesMenu()
         {
             var player = _playerManager.LocalSession?.AttachedEntity;
             if (emote.Category == EmoteCategory.Invalid ||
-                emote.ChatTriggers.Count == 0 ||
+                emote.ChatTriggers.Count == 0 && emote.Category != EmoteCategory.GlobalCustom ||
```

Comment:
Make a new boolean that controls this "can be triggered via menu" behavior instead.